### PR TITLE
the-stream-team

### PR DESCRIPTION
**Pull request by chris-short**
I do this because I keep deleting my local copy. I'm a genius.

Signed-off-by: Chris Short <chris@chrisshort.net>

### CODE REVIEW

File: communication/youtube/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+The Kubernetes community is an open community. As such, numerous meetings are
+recorded and shared through the
+[Kubernetes YouTube](https://www.youtube.com/channel/UCZ2bu0qutTOM0tHYa_jkIwg)
+channel. Additionally, requests to live stream meetings are increasing as the
+community grows.
+
+## YouTube Channel Guidelines
+
+Please refer to [YouTube Channel Guidelines] with questions regarding YouTube.
```
Comment (mrbobbytables) [inline communication/youtube/README.md]: Missing the reference link at the bottom. 
```
[youtube channel guidelines]: ./youtube-guidelines.md
```

Having that at the bottom of the doc will fix it for both references to `[YouTube Channel Guidelines]` 👍

File: communication/youtube/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# YouTube Management Project (The Stream Team)
```
Comment (mrbobbytables) [inline communication/youtube/README.md]:
```suggestion
# YouTube Management Project
```

Stream team is fun, but I would remove it from the heading for link purposes.

File: communication/youtube/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## YouTube Channel Guidelines
+
+Please refer to [YouTube Channel Guidelines] with questions regarding YouTube.
+
+## Intent of Subproject
+
+This subproject's purpose is intended to be:
+
+1. Appropriately prioritize YouTube as a Kubernetes community resource and means
```
Comment (mrbobbytables) [inline communication/youtube/README.md]: Small nit on the list items 😬 As they are complete sentences, would you mind adding a period at the end?

**Comment by chris-short** [comment]
Double-check me please, @mrbobbytables, but I think it's good.

**Comment by mrbobbytables** [comment]
thanks @chris-short !

/lgtm

**Pull request merged**